docs/radar.js

```diff
@@ -182,23 +182,17 @@ function radar_visualization(config) {
     return "translate(" + x + "," + y + ")";
   }
 
-  function displayText(d) {
-    if (d.hasOwnProperty("link")) {
-      return d.label + " 🔗";
```

Comment:
This removes the functionality to visualise that an element has a link set which I would rather keep